src/pages/stage-page/validate-stage-page.ts

```diff
@@ -23,9 +23,10 @@ export const validateTimetableStage = ({
 
   const errors: ValidationErrorItem[] = [];
 
-  const startEventValidation = eventSchema.validate(
+  const startEventValidation = stageSchema.validate(
     {
-      eventDate: startEvent.eventDate,
+      startDate: startEvent.eventDate,
+      endDate: endEvent?.eventDate,
       notes: startEvent.notes,
     },
     { abortEarly: false }
```

Comment:
in the path down on line 39 I would use the name of the stage rather than the eventKey:
`
path: [stageName, ...error.path],
`
stage name should be available through formProps (this also require changing in the Stage component render where we get the error for the form fields)

Reply:
that will make the error id read as follow: "publication-stage-startDate" rather than "publication-start-startDate"